Containers/postgresql/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # From https://github.com/docker-library/postgres/blob/master/13/alpine/Dockerfile
-FROM postgres:14.6-alpine
+FROM postgres:15.1-alpine3.16
```

Comment:
and wait for alpine 3.17.1?

```suggestion
FROM postgres:15.1-alpine
```

Reply:
```suggestion
FROM 15.1-alpine3.17
```
maybe pin the alpine version?

Reply:
thought about this as well, however there was a time when minor posgres updates where not pushed with older alpine version anymore. That is why I removed that. Also it is pretty unlikely that alpine updates break this specific container.

Reply:
removed that again.